src/test/kotlin/org/rust/ide/navigation/goto/RustGotoSuperHandlerTest.kt

```diff
@@ -0,0 +1,20 @@
+package org.rust.ide.navigation.goto
+
+import com.intellij.lang.CodeInsightActions
+import org.rust.lang.RustLanguage
+import org.rust.lang.RustTestCaseBase
+
+class RustGotoSuperHandlerTest : RustTestCaseBase() {
+    override val dataPath = "org/rust/ide/navigation/fixtures/goto_super"
+
+    fun testGotoSuper() {
+        myFixture.copyFileToProject("main.rs", "/main.rs")
```

Comment:
It indeed is a bit strange that during a test we will have a target pointing nowhere, but it should be OK.